.github/workflows/docs.yml

```diff
@@ -19,8 +18,17 @@ jobs:
         with:
           key: ${{ github.ref }}
           path: .cache
```

Comment:
I don't know what we're caching here, I don't see `.cache` locally, but I just left it since I wasn't sure. If anyone knows why it's here, let me know.

Reply:
I'm just going to remove it, I see this in the action logs:
```
Post job cleanup.
Warning: Path Validation Error: Path(s) specified in the action for caching do(es) not exist, hence no cache is being saved.
```

So it's clearly not being used.